src/tablist.js

```diff
@@ -18,6 +18,10 @@ SideTabList.prototype = {
     if (this.alwaysShrink) {
       this.maybeShrinkTabs();
     }
+
+    this.useRegExpFilter = (await browser.storage.local.get({
```

Comment:
Your other PR has been merged actually (https://github.com/eoger/tabcenter-redux/commit/5f4de25c63e7e7bb057601d3067bc31f03730850), you can rebase against master ;-)